src/common/runtime/wpt.ts

```diff
@@ -1,63 +1,59 @@
+// Implements the wpt-embedded test runner (see also: wpt/cts.html).
+
 import { DefaultTestFileLoader } from '../framework/file_loader.js';
 import { Logger } from '../framework/logging/logger.js';
 import { parseQuery } from '../framework/query/parseQuery.js';
-import { TestTreeLeaf } from '../framework/tree.js';
-import { AsyncMutex } from '../framework/util/async_mutex.js';
```

Comment:
Is this the only usage of `AsyncMutex`? Can we remove it?

Reply:
I think it is, and I was thinking about removing it but forgot. May as well do it now.